openc3-minio/Dockerfile

```diff
@@ -11,13 +11,6 @@ ENV CURL_CA_BUNDLE=/devel/cacert.pem
 ENV REQUESTS_CA_BUNDLE=/devel/cacert.pem
 ENV NODE_EXTRA_CA_CERTS=/devel/cacert.pem
 
-# Update packages to eliminate CVEs if we're on docker.io (not ironbank)
-RUN if [[ $OPENC3_DEPENDENCY_REGISTRY == 'docker.io' ]]; then \
-  microdnf update --nodocs -y && microdnf clean all; \
-  groupadd -g 1001 minio; \
-  useradd -r -u 1001 -m -g minio minio; \
-  fi
-
```

Comment:
The new minio containers are now based on ubi-micro (https://github.com/minio/minio/issues/18473) which do not include a package manager. They also do not include any additional tools. So there's no packages to update and I think we can just run as USER 1001 wtihout adding the user / group (see https://github.com/minio/minio/issues/18380).